app/controllers/consultations_controller.rb

```diff
@@ -3,16 +3,4 @@ def index
     filter_params = params.except(:controller, :action, :format, :_)
     redirect_to publications_path(filter_params.merge(publication_filter_option: 'consultations'))
```

Comment:
Could we use a prefix route? `/government/consultations`. That might let the filter params pass through to Whitehall with the explicit routes taking the actual consultation requests to government frontend. Need to check that though as I'm not sure.